nemo/collections/asr/models/rnnt_models.py

```diff
@@ -120,6 +123,32 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             self._optim_variational_noise_std = 0
             self._optim_variational_noise_start = 0
 
+        # Setup normalized gradients for model
+        if 'normalize_joint' in self.cfg:
```

Comment:
Can they be done like this?
self._normalize_joint = self.cfg.get(normalize_joint, False)